Turn debug mode off by default.

cmurphy opened the pull request:
Now run in production mode with PUPPETANALYTICS_SETTINGS=production_settings.py python puppetanalytics/app.py

nibalizer commented on puppetanalytics/app.py:
```diff
@@ -12,6 +12,7 @@
 
 @app.route("/")
 def mainpage():
+    a=b
```
This should go away.

cmurphy replied:
lol woops

nibalizer commented on puppetanalytics/config.py:
```diff
@@ -0,0 +1,8 @@
+class Config(object):
```
Can we move the stuff from settings.py in as well?

cmurphy replied:
There is no settings.py. This would be the beginning of settings configuration. We could rename it to settings.

greghaynes commented on puppetanalytics/app.py:
```diff
@@ -131,6 +132,10 @@ def init_database():
 
 
 if __name__ == "__main__":
+    import os
```
Imports should be at top of file

greghaynes commented on puppetanalytics/app.py:
```diff
@@ -131,6 +132,10 @@ def init_database():
+    import os
     init_database()
-    app.debug = True
+    if os.environ.get('PUPPETANALYTICS_DEBUG') == 'true':
```
We need a truthy method for this. If one doesnt already exist (I dont think weve made it yet) we should make one.

greghaynes replied:
Higher level - I dont think the app should automagically pick the correct config file. The operator should specify a config and we should respect their decision. Why not just ship a production config and devleopment config, and let the user specify?

cmurphy committed:
Add settings classes

Default mode is production mode. Run in development mode with
PUPPETANALYTICS_SETTINGS=puppetanalytics.settings.DevelopmentSettings python puppetanalytics/app.py
